Hush/Assets/Scripts/Game/SceneManager.cs

```diff
@@ -7,14 +8,14 @@ public class SceneManager : MonoBehaviour
     {
         public static SceneManager Instance;
 
-        [Header("Scene Build Indexes")]
-        [SerializeField] private int[] levelSceneIndexes;
-        [SerializeField] private int mainMenuIndex;
-        [SerializeField] private int finalSceneIndex;
-        [SerializeField] private int endgameSceneIndex;
-        
-        [Header("New Game Parameters")]
-        [SerializeField] private SpellType initialLightSpell = SpellType.FireballSpell;
+        [Header("Scene Build Indexes")] [SerializeField]
+        private Scenes[] levelSceneIndexes;
+
+        [Header("New Game Parameters")] [SerializeField]
+        private SpellType initialLightSpell = SpellType.FireballSpell;
+
+        [SerializeField] private Scenes firstFloor = Scenes.FloorA;
```

Comment:
Assument the first floor is simply the first entry in levelSceneIndexes

Reply:
Well, it's the first floor you want to load in. But for testing purposes, I was setting the "default first floor" to one of the other ones to make sure they were okay

Reply:
So this is just a testing value